src/components/modals/Modal.vue

```diff
@@ -77,6 +83,32 @@ export default {
       color: red;
     }
   }
+  .maximize-toggle {
+    position: absolute;
+    right: 55px;
+    top: 20px;
+    cursor: pointer;
+    &:hover {
+      color: red;
+    }
+  }
+}
+
+.maximized {
```

Comment:
If I maximize a modal that has very little content (like the edition modal), the content will stick to the top of the modal. I think it looks better to vertically center it, like so:
```css
    display: flex;
    align-content: center;
    align-items: center;
    justify-content: center;
```
❗ This would require adding another generic container in the Modal template though, otherwise some modals are going to look bad.

Reply:
Yeah, the small modals are a pain here!

Reply:
`align-items: center;`
This seems to break the alignment of the columns headers in the character reference.

`justify-content: center;`
This breaks the scrolling when the modal is bigger than the window, which happens for the character reference with e.g. S&V.

But, with just the other two remaining, I think it all works and looks fine.